### DISCUSSION TITLE
SOS landing page

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1616] [Unsupported Versions] [answered]
Update 2.3.0 -> 2.3.1

**Comment by TOoSmOotH** [comment]
The SOC page should be displaying 2.3.1 based on the code. You might need to force refresh after the upgrade.

**Comment by darth-drew** [comment]
Sorry, I meant the security onion solutions web page.

Andrew R. Albertson
C: (540)908-8551

**Comment by dougburks** [comment, marked as answer]
https://securityonionsolutions.com/software/ has been updated!
